packages/core/logging/core-logging-server-internal/BUILD.bazel

```diff
@@ -37,6 +37,7 @@ TYPES_DEPS = [
   "@npm//@types/node",
   "@npm//@types/jest",
   "@npm//@types/lodash",
+  "@npm//rxjs",
```

Comment:
Yeah, I'm also surprised but there's a change that the old version of this task pulled RxJS into node_modules as a byproduct of some other dependency, not quite sure though. In the future dependencies will be determined automatically from the source so it will be less of a mystery 😊